fvm/evm/types/chainIDs.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import (
+	"math/big"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+var (
+	FlowEVMTestnetChainID = big.NewInt(646)
```

Comment:
nit: why is Flow in the name, wouldn't this just be `EVMTestnetID` and `EVMMainnetID`

Reply:
We could name it EVMTestnetID , I used the Flow in case someone externally is using it and we use name FlowEVM for the evm chain